common/src/main/java/com/wynntils/features/redirects/TerritoryMessageRedirectFeature.java

```diff
@@ -54,8 +54,6 @@ public void onSubtitleSetText(SubtitleSetTextEvent event) {
     // text event.
     @SubscribeEvent
     public void onChat(ChatMessageReceivedEvent event) {
-        if (event.getOriginalCodedMessage()
-                .getMatcher(TERRITORY_MESSAGE_PATTERN)
-                .matches()) event.setCanceled(true);
+        if (event.getOriginalCodedString().getMatcher(TERRITORY_MESSAGE_PATTERN).matches()) event.setCanceled(true);
```

Comment:
I realize this is not new, but since you touch this anyway. Please add braces to the if statement.

Reply:
Or, whatever, I can fix this separately. Better get this in since other PRs depend on it.